cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/management-cluster-settings-step/ManagementClusterSettings.tsx

```diff
@@ -106,7 +107,7 @@ function ManagementClusterSettings(props: Partial<MCSettings>) {
                             <label>Select a control plane node profile</label>
                             {nodeProfiles.map((nodeProfile, index) => {
                                 return (
-                                    <CdsRadio cds-layout="m:lg m-l:xl p-b:sm" key={index}>
+                                    <CdsRadio cds-layout="m:lg m-l:xl p-b:sm" key={index} data-testid="cds-radio">
```

Comment:
I'm guessing it's my oversight, but I'm not seeing where this `data-testid` is being used

Reply:
we need it to get the element in the unit test.